source/runner/Executor.ts

```diff
@@ -148,11 +148,40 @@ export class Executor {
       this.d("Writing to STDOUT:", results)
       // Human-readable format
 
+      const tick = chalk.bold.greenBright("✓")
+      const cross = chalk.bold.redBright("ⅹ")
+      let output = ""
+
+      if (fails.length > 0) {
+        const s = fails.length === 1 ? "" : "s"
+        const are = fails.length === 1 ? "is" : "are"
+        const message = chalk.underline.red("Failing the build")
+        output = `Danger: ${cross} ${message}, there ${are} ${fails.length} fail${s}.`
+        process.exitCode = 1
+      } else if (warnings.length > 0) {
+        const message = chalk.underline("not failing the build")
+        output = `Danger: ${tick} found only warnings, ${message}`
+      } else if (messages.length > 0) {
+        output = `Danger: ${tick} passed, found only messages.`
+      } else if (!messages.length && !fails.length && !messages.length && !warnings.length) {
+        output = `Danger: ${tick} passed review, received no feedback.`
+      }
+
+      const allMessages = [...fails, ...warnings, ...messages, ...markdowns].map(m => m.message)
+      const oneMessage = allMessages.join("\n")
+      const longMessage = oneMessage.split("\n").length > 30
```

Comment:
My laptop full screen is 40 lines, so this is 3/4 of that

Reply:
cool 👍